app.js

```diff
@@ -1,4 +1,29 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+var express = require('express');
+var app = express();
+var trucks = {
+	'Athenas': 'Meditteranean', 'Beanfish': 'Asian', 'Marination': 'Hawaaian-Korean'
+};
+app.use(express.static('public'));
+
+app.get('/trucks/:name', function (request, response) {
+	var truck = request.params.name;
+	var foodType = trucks[request.params.name];
+
+	// capitalize first letter, make everything else lowercase
+	var foodTruck = truck[0].toUpperCase + truck.slice(1).toLowerCase();
```

Comment:
This line of code is unnecessary. You don't use foodTruck after you assign something to it, so I would leave this out.